tests/integration_test/firebase_auth/firebase_auth_multi_factor_e2e_test.dart

```diff
@@ -170,6 +170,118 @@ void main() {
+              Completer completer = Completer<String>();
+
+              unawaited(
+                FirebaseAuth.instance.verifyPhoneNumber(
+                  phoneNumber: testPhoneNumber,
+                  multiFactorSession: session,
+                  verificationCompleted: (PhoneAuthCredential credential) {
+                    if (!completer.isCompleted) {
+                      return completer.completeError(
+                        Exception(
+                          'verificationCompleted should not have been called',
+                        ),
+                      );
+                    }
+                  },
+                  verificationFailed: (FirebaseException e) {
+                    if (!completer.isCompleted) {
+                      return completer.completeError(
+                        Exception(
+                          'verificationFailed should not have been called',
+                        ),
+                      );
+                    }
+                  },
+                  codeSent: (String verificationId, int? resetToken) {
+                    completer.complete(verificationId);
+                  },
+                  codeAutoRetrievalTimeout: (String foo) {
+                    if (!completer.isCompleted) {
+                      return completer.completeError(
+                        Exception(
+                          'codeAutoRetrievalTimeout should not have been called',
+                        ),
+                      );
+                    }
+                  },
+                ),
+              );
+
+              return completer.future as FutureOr<String>;
+            }
+
+            final verificationId = await getCredential();
+
+            final smsCode = await emulatorPhoneVerificationCode(
+              testPhoneNumber,
+            );
+
+            final credential = PhoneAuthProvider.credential(
+              verificationId: verificationId,
+              smsCode: smsCode!,
+            );
+
+            expect(credential, isA<PhoneAuthCredential>());
+
+            await user.multiFactor.enroll(
+              PhoneMultiFactorGenerator.getAssertion(
+                credential,
+              ),
+              displayName: 'My phone number',
+            );
+
+            final enrolledFactors = await multiFactor.getEnrolledFactors();
+
+            // Assertions
+            expect(enrolledFactors.length, 1);
+            expect(enrolledFactors.first.displayName, 'My phone number');
+
+            try {
+              await user.multiFactor.unenroll(
+                factorUid: 'unknown',
+              );
+            } catch (e) {
+              expect(e, isA<FirebaseAuthException>());
+            }
+
+            final enrolledFactorsAfter = await multiFactor.getEnrolledFactors();
+
+            // Assertions
+            expect(enrolledFactorsAfter.length, 1);
+          },
+          skip: kIsWeb || defaultTargetPlatform != TargetPlatform.android,
```

Comment:
Should this also be true for iOS?

Reply:
This test (from which this one is copied) was already skipped for iOS. We cannot get the multifactor to work on iOS in CI because it tries to verify for Recaptcha and fails :/

Reply:
Perhaps leave a note explaining such 😄

Reply:
Good call I'll add that :)